test/test_jit.py

```diff
@@ -1188,6 +1188,36 @@ def forward(self, x):
                        .check("return") \
                        .run(str(get_module_method(m, 'conv', 'conv2d_forward').graph))
 
+    def test_insert_quant_dequant_multi_uses(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+
+            def forward(self, x, w0, w1, w2):
+                a = F.conv2d(x, w0)
+                b = F.conv2d(a, w1)
+                c = F.conv2d(a, w2)
+                return b + c
+
+        m = torch.jit.script(M())
+        observer = torch.jit.script(Observer())
+        qconfig_dict = {
+            '':
+            QConfig(
+                activation=observer._c,
+                weight=observer._c)
+        }
+        torch._C._jit_pass_insert_observers(m._c, "forward", qconfig_dict, True)
+        data = torch.randn(1, 3, 10, 10, dtype=torch.float)
+        weight = torch.randn(3, 3, 3, 3, dtype=torch.float)
+
+        get_forward(m._c)(data, weight, weight, weight)
```

Comment:
Nit: you could just do `m(data, weight, weight, weight)` here without extracting the forward method.

Reply:
We may not even need the data for calibration since we just test the quant and dequant insertion.

Reply:
we need to run calibration before inserting quant/dequant

Reply:
We can just use the default qparams?